### ISSUE TITLE
Page editor: an option for a fixed width

### ISSUE THREAD

**Issue by th0rgall** [getgrav/grav-plugin-admin#1159]
[Feature request] I like the distraction-free writing mode, but I find it annoying to read/write long lines of text. On my 16:9 1080p monitor, this is how the mode looks:
![screenshot from 2017-07-15 10-28-17](https://user-images.githubusercontent.com/11543641/28237961-edf1bb8c-6949-11e7-805b-3b0869a3e4a0.png)

What I'd like to have is something more like this, which was achieved by adding a padding property to an appropriate HTML element locally.

![screenshot from 2017-07-15 10-35-37](https://user-images.githubusercontent.com/11543641/28237977-37bfcace-694a-11e7-86f4-f97fa0ba7a83.png)

I looked around in the editor toolbar, admin panel settings and issues here, but couldn't really find anything relating to this.

It would be nice to have an option in the admin panel settings to fix the width of the content editor (eg. to 760px), and/or a button in the editor toolbar to switch to a more readable format.

**Comment by flaviocopes** [comment]
I agree it's better have a max-width on the editor content by default. You can achieve it now by using https://github.com/RomainFallet/grav-plugin-customadmin and add

```
.grav-editor-fullscreen { background-color: white; }
.grav-editor-fullscreen .grav-editor-content { max-width: 800px; margin: 0 auto!important; }
```

**Comment by th0rgall** [comment]
@flaviocopes this works like a charm, thank you!
